src/app/Modal/DeleteWarningModal.tsx

```diff
@@ -68,7 +68,7 @@ export const DeleteWarningModal = ({ onAccept, onClose, ...props }: DeleteWarnin
         <Stack hasGutter key="modal-footer-stack">
           <Split key="modal-footer-split">
             <Button variant="danger" onClick={onAcceptClose}>
-              Delete
+              {props.warningType === DeleteOrDisableWarningType.ClearAllDashboardCards ? 'Clear' : 'Delete'}
```

Comment:
This interface should receive a prop called `acceptTitle` and place it here (default to `Delete`).

https://github.com/cryostatio/cryostat-web/pull/1078#discussion_r1294879384

Reply:
This gives control to the parent component. If we do checks as of now, we have to update this component every time a new text is required.